src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -96,8 +96,14 @@ private static bool IsCursorOutsideOfExtent(IScriptPosition cursor, IScriptExten
             return cursor.Offset < extent.StartOffset || cursor.Offset > extent.EndOffset;
         }
 
-        internal CompletionContext CreateCompletionContext(ExecutionContext executionContext)
+        internal CompletionContext CreateCompletionContext(CompletionContext completionContext)
```

Comment:
The signature suggests that it should be some sort of a copy-ctor, but it's not.